plutus-core-spec/Makefile

```diff
@@ -0,0 +1,37 @@
+DOC=plutus-core-specification
+
+PDF=${DOC}.pdf
+BIB=${DOC}.bib
+
+FIGS=./figures
+
+SRC = ${DOC}.tex ${BIB} ${FIGS}/*.tex 
+
+
+LATEX = pdflatex -halt-on-error -shell-escape # To get pstricks to work with PDF
+BIBTEX = bibtex
+
+.PHONEY: all pdf figs again clean 
+
+#----------------------------------------------------------------
+
+all: pdf
+
+pdf: ${PDF}
+
+${DOC}.pdf: ${SRC}
+	${LATEX}  ${DOC}
+	${BIBTEX} ${DOC}
+	${LATEX}  ${DOC}   # to make sure the (cross)references are correct
+	${LATEX}  ${DOC}
```

Comment:
I *highly* recommend ditching the whole makefile and using `latexmk`, which handles all this stuff for you and is very battle-tested.

Reply:
Well yeah, but this is pretty simple.  latexmk takes a lot longer and produces vast amounts of crap.  I'm not going to demand that other people don't use latexmk though.

Reply:
If latexmk is doing more work than your makefile then I expect that your makefile is wrong :)